pkg/ibmvpc/examples/acl_testing5subnetsDiff.md

```diff
@@ -0,0 +1,12 @@
+# Diff between VPC test-vpc-ky1 and VPC test-vpc-ky2
+## Subnets diff report
+| type | src |  dst | conn1 | conn2 | diff-info |
```

Comment:
the column `diff-info` should be `subnets-diff-info` or `vsis-diff-info` as in the `txt ` format

Reply:
done